libobs/obs-windows.c

```diff
@@ -64,13 +64,42 @@ char *find_libobs_data_file(const char *file)
 	struct dstr path;
 	dstr_init(&path);
 
+	WCHAR *env_path = 0;
+	DWORD env_path_sz = 
+		GetEnvironmentVariableW(L"OBS_DATA_PATH", NULL, 0);
+
+	env_path = bmalloc(env_path_sz * sizeof(WCHAR));
```

Comment:
Minor/insignificant, but you should probably use wchar_t rather than WCHAR to be more consistent with the rest of the file.  (Yea they're the same, it just feels strange to use different type names.  It also affects highlighting in certain editors.)_